keras/models.py

```diff
@@ -790,40 +790,78 @@ def compile(self, optimizer, loss,
         self.sample_weights = self.model.sample_weights
         self.targets = self.model.targets
 
-    def fit(self, x, y, batch_size=32, epochs=10, verbose=1, callbacks=None,
-            validation_split=0., validation_data=None, shuffle=True,
-            class_weight=None, sample_weight=None, initial_epoch=0, **kwargs):
+    def fit(self, x=None,
```

Comment:
I copied and pasted from the fit function of `training.py`. After adding parameters the long lines were awkward.